spec/fixtures/feeds.yml

```diff
@@ -0,0 +1,4 @@
+fashion_tumblr:
+  name: Fashion Tumblr
+  slug: fashion_tumblr
```

Comment:
If auto-generated the slug would be "fashion-tumblr", so should probably be that in the fixture too.